src/features/events/repo/EventsRepo.ts

```diff
@@ -77,10 +77,7 @@ export default class EventsRepo {
   async addParticipant(orgId: number, eventId: number, personId: number) {
     const participant = await this._apiClient.put<ZetkinEventParticipant>(
       `/api/orgs/${orgId}/actions/${eventId}/participants/${personId}`,
-      {
-        id: personId,
-        reminder_sent: null,
-      }
+      {}
```

Comment:
Is it possible to pass `null`?

Reply:
I don't think so, it doesn't want a body at all. And what would be null? `id` can't be.

Reply:
I mean passing `null` instead of an empty object, like this:

```suggestion
      null
```

But looking at the type signature for `put()`, what I'm looking for is really omitting the data argument altogether, i.e:

```suggestion
```